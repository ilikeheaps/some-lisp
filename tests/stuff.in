(+ 1 2)
(lambda (x) (+ 1 2))
((lambda (x) (+ 1 x)) 2)
(((lambda (x) (lambda (t) (+ x t))) 1) 2)
exit