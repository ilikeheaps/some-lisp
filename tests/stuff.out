3
(lambda (x) (<haskell function> 1 2))
3
3
